fcrepo-ldpath/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+       xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+       xsi:schemaLocation="
+       http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0 http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+       http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0 http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.2.xsd
+       http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
+
+   <!-- system environment property placeholders -->
+   <ext:property-placeholder placeholder-prefix="$[" placeholder-suffix="]" />
+
+   <!-- OSGI blueprint property placeholder -->
+   <cm:property-placeholder id="properties" persistent-id="org.fcrepo.camel.ldpath" update-strategy="reload">
+     <cm:default-properties>
+       <cm:property name="fcrepo.authUsername" value=""/>
+       <cm:property name="fcrepo.authPassword" value=""/>
+       <cm:property name="fcrepo.authHostname" value="localhost"/>
+       <cm:property name="fcrepo.authPort" value="8080"/>
+       <cm:property name="fcrepo.baseUrl" value="http://localhost:8080/fcrepo/rest"/>
+       <cm:property name="fcrepo.cache.timeout" value="0"/>
+       <cm:property name="rest.prefix" value="/ldpath"/>
+       <cm:property name="rest.host" value="localhost"/>
+       <cm:property name="rest.port" value="9086"/>
+       <cm:property name="cache.dir" value="$[karaf.data]/ldcache"/>
```

Comment:
When I build `fcrepo-camel-toolbox`, a directory named `$[karaf.data]` is created within the `fcrepo-ldpath` directory... which is undesirable.